tests/foreman/api/test_host.py

```diff
@@ -391,45 +467,59 @@ def test_positive_create_with_compresource(self):
         host = entities.Host(
             compute_resource=compresource, location=loc, organization=org
         ).create()
-        self.assertEqual(host.compute_resource.read().name, compresource.name)
+        assert host.compute_resource.read().name == compresource.name
+        new_compresource = entities.LibvirtComputeResource(
+            location=[host.location], organization=[host.organization]
+        ).create()
+        host.compute_resource = new_compresource
+        host = host.update(['compute_resource'])
+        assert host.compute_resource.read().name == new_compresource.name
 
     @tier2
-    def test_positive_create_with_model(self):
-        """Create a host with model specified
+    def test_positive_create_and_update_with_model(self):
+        """Create and update a host with model specified
 
         :id: 7a912a19-71e4-4843-87fd-bab98c156f4a
 
-        :expectedresults: A host is created with expected model assigned
+        :expectedresults: A host is created and updated with expected model assigned
 
         :CaseLevel: Integration
         """
         model = entities.Model().create()
         host = entities.Host(model=model).create()
-        self.assertEqual(host.model.read().name, model.name)
+        assert host.model.read().name == model.name
+        new_model = entities.Model().create()
```

Comment:
check if same available in global fixtures else worth creating there ?

Reply:
DONE.